src/pages/about.js

```diff
@@ -24,10 +24,24 @@ module.exports = React.createClass({
       aboutCopy: aboutCopy
     });
   },
+  imageTest: function() {
+    if (this.props.test === "img-a") {
+      return (<picture>
+        <source srcSet="/assets/images/homepage-images/teacher-wide.jpg" media="(max-width: 800px)" />
+        <img className="homepage-image" src="/assets/images/homepage-images/teacher-full.jpg" alt="Man pointing at student's computer screen"/>
+      </picture> );
+    } else if (this.props.test === "img-b") {
```

Comment:
you don't need `else` if you return in the previous conditional.
```
if (x) { return blah; }
if (y) { return blah2; }
return finalThing
```
Additionally, it'll be good to cache that prop val in a variable
```
let test = this.props.test;
if (test === "img-a") {
   return ...;
}
if (test === "img-b" {
  return ... ;
}
return ...;
```